src/drive/mobile/containers/DriveMobileRouter.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
```

Comment:
This file binds the Authentication module (which is application-agnostic) and the Cozy Drive app.